lib/rails_admin_import/formats/dummy_importer.rb

```diff
@@ -0,0 +1,16 @@
+module RailsAdminImport
+  module Formats
+    class DummyImporter
+      def initialize(*)
+      end
+
+      def valid?
+        false
+      end
+
+      def error
+        I18n.t('admin.import.invalid_format')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.